internal/http/testcomposer.go

```diff
@@ -164,12 +164,12 @@ func (c *TestComposer) Frameworks(ctx context.Context) ([]framework.Framework, e
 	if err = c.doJSONResponse(req, 200, &resp); err != nil {
 		return []framework.Framework{}, err
 	}
-	return resp, nil
+	return uniqFrameworkSet(resp), nil
```

Comment:
I think it'd be great to use this opportunity to change the return type to a string slice. That's all the initializer (the user of this method) really cares about.

Reply:
That's simplified now :-)